api/envoy/service/ratelimit/v3/rls.proto

```diff
@@ -110,6 +110,9 @@ message RateLimitResponse {
 
     // The limit remaining in the current time unit.
     uint32 limit_remaining = 3;
+
+    // Number of seconds until reset of the current limit window.
+    uint32 seconds_until_reset = 4;
```

Comment:
WDYT about using a protobuf duration here? I think that would future proof it a bit better?